### ISSUE TITLE
Cant install jqmthemer

### ISSUE THREAD

**Issue by Nears** [pimatic/jqmthemer#1]
I can't install jqmthemer when i try to install the pimatic-mobile-frontend.

The error says that it is not i the npm regestry.
I saw you were working i this in the last month. 

Can you please help. Thanks.

Update: I was working on it and now i think you must only publish it to npm ? 
Am I correct? Just trying to learn..... :-)

**Comment by sweetpi** [comment]
Yes, you are right. You can also install it directly from the github repository: `npm install pimatic/jqmthemer`. I didn't publish it yet because I had no time to write the README :)

**Comment by sweetpi** [comment]
Published it. But what error did you get?

**Comment by Nears** [comment]
Thanks for not publishing it before :-)
I learned a lot new things while trying to install it on my pi. I had a lot of problems because im not used to npm yet. But I finally made it by downloading it and install it locally. Did not know about the way you mentioned above.

The error came up when i tried to install pimatic-mobile-frontend and the installation was aborted. It says that jqmthemer was not found and that i should contact the developer in order to publish it.

Btw: I like your pimatic project. Maybe later on I hope I can contribute to this project :-)

**Comment by Nears** [comment]
Hi, sorry I must reopen this issue.

Today a performed an update and now Npm tells me that there is no matching version found for jqmthemer. Is it because the package.json of mobile-frontend plugin says:

"dependencies": {
.....
"jqmthemer": "pimatic/jqmthemer",
....

??

Thanks again for your help... :-)

**Comment by sweetpi** [comment]
What does

```
 npm --version
```

say?

**Comment by Nears** [comment]
1.3.21

Just to be sure you got me right. The update i made was not a system update but a pimatic update via the mobile-frontend.
